src/Text/LLVM/PP.hs

```diff
@@ -667,6 +668,18 @@ ppCall tc ty f args
   body = "call" <+> ppCallSym ty f
       <> parens (commas (map (ppTyped ppValue) args))
 
+ppCallBr :: LLVM => Type -> Value -> [Typed Value] -> BlockLabel -> [BlockLabel] -> Doc
+ppCallBr ty f args to indirectDests =
+  "callbr"
+     <+> ppType res <+> ppValue f <> parens (commas (map (ppTyped ppValue) args))
+     <+> "to" <+> ppLab to <+> brackets (commas (map ppLab indirectDests))
```

Comment:
The example [here](https://github.com/llvm/llvm-project/commit/784929d0454c4df6a98ef6fbbd1d30a6f71f9c16#diff-1d1772820f1edf0653335111c7ab690c53a2df20887eadadf55d7a9c17aa907aR6851-R6852) includes an `or jump [labels]` bit. Does this code print out the `or jump` part? (`llvm-pretty-bc-parser`'s CI would catch this, I think, but it hasn't run on GaloisInc/llvm-pretty-bc-parser#187 yet.)

Reply:
Hmm, interesting. The current Language Reference doesn't have that syntax: https://llvm.org/docs/LangRef.html#callbr-instruction.
```
<result> = callbr [cconv] [ret attrs] [addrspace(<num>)] <ty>|<fnty> <fnptrval>(<function args>) [fn attrs]
              [operand bundles] to label <fallthrough label> [indirect labels]
```
I also did run the disassembly tests on the testcase in the bitcode parser PR and they passed, but that was probably with whatever version of LLVM I have installed and got run in the tests.

Reply:
Ah, good catch. The link that I gave above is the syntax used in LLVM 9.0, but it changed in LLVM 10.0 in llvm/llvm-project@93d87260f1f7b5ec6775fe88891e47a57e6392d6. I'm not quite sure if there is a good way to handle the difference in LLVM versions here, but I'm definitely more inclined to pick the modern syntax.

Reply:
Done.

Reply:
Here are the disasm tests passing on the bitcode parser in CI, for good measure: https://github.com/GaloisInc/llvm-pretty-bc-parser/runs/6233698029?check_suite_focus=true#step:7:29